### DISCUSSION TITLE
Memcached TLS support

### DISCUSSION THREAD

**Discussion by gator2003** [OpenIDC/mod_auth_openidc#1025] [Q&A] [answered]
I was looking through the repo and didn't see anything about using a secure memcache connection (TLS). Is this possible in the current release? When I enable TLS on memcached, httpd still connects for non-OIDC connections, but the mod doesn't establish the connection.

**Comment by zandbelt** [comment, marked as answer]
no, connecting over TLS to memcache is not possible from mod_auth_openidc as it uses apr_memcache which has no support for that; you could run stunnel in between as a workaround

**Reply by gator2003** [reply]
Understood, thank you!
